arelle/CntlrWinTooltip.py

```diff
@@ -38,6 +38,8 @@
 # Ideas gleaned from PySol
 
 import tkinter
+from tkhtmlview import HTMLLabel, HTMLText
```

Comment:
`tkhtmlview==0.1.3` needs to be included in requirements.txt as a dep for the build